packages/atomic/src/components/common/layout/item-layout.ts

```diff
@@ -0,0 +1,62 @@
+import {
+  getItemDisplayClasses,
+  type ItemDisplayDensity,
+  type ItemDisplayImageSize,
+  type ItemDisplayLayout,
+} from './display-options';
+
+import {
+  containsSections,
+  type ItemSectionTagName,
+} from './item-layout-sections';
+
+export class ItemLayout {
```

Comment:
I'm not sure why this has to be a class... it seems to me like it could be refactored into a pure function.